next.config.js

```diff
@@ -61,4 +63,6 @@ const sentryWebpackPluginOptions = {
   // https://github.com/getsentry/sentry-webpack-plugin#options.
 };
 
-module.exports = withSentryConfig(moduleExports, sentryWebpackPluginOptions);
+const plugins = [withRoutes, (config) => withSentryConfig(config, sentryWebpackPluginOptions)];
+
+module.exports = () => plugins.reduce((config, plugin) => plugin(config), nextConfig);
```

Comment:
Followed the thread here - https://github.com/cyrilwanner/next-compose-plugins/issues/59